server/routes/adjudicationForReport/scheduleHearing/scheduleHearingValidation.ts

```diff
@@ -3,7 +3,8 @@ import { formatDate, formatTimestampToDate, formatTimestampToTime } from '../../
 
 type ScheduleHearingForm = {
   hearingDate?: SubmittedDateTime
-  locationId?: number
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  locationId?: any
```

Comment:
is any correct here?

Reply:
My mistake it should be string only. 
The ScheduleHearingForm is used in the validation of `/schedule-hearing/:chargeNumber` when posting. 
The select drop-down on this page now gets its entries from the locations-api which will have UUIDs for the locationId